API/Repositories/AppointmentRepository.cs

```diff
@@ -133,7 +142,7 @@ public async Task<SuggestionsDto> GetSuggestionsAsync(int doctorId)
                 .Select(a => new Suggestion
                 {
                     value = a.Id,
-                    label = a.ScheduledDate.ToUniversalTime() + ", " + a.Patient.FirstName + " " +
+                    label = /*a.ScheduledDate.ToUniversalTime()*/ TimeZoneInfo.ConvertTimeFromUtc(a.ScheduledDate,TimeZoneInfo.Local) + ", " + a.Patient.FirstName + " " +
                             a.Patient.LastName
```

Comment:
Komentarz chyba można wyrzucić, a stringi się całkiem fajnie łączy w taki sposób (dość podobny do JSa)
```suggestion
                    label = $"{TimeZoneInfo.ConvertTimeFromUtc(a.ScheduledDate,TimeZoneInfo.Local)}, {a.Patient.FirstName} {a.Patient.LastName}";
```